app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/agenda_screen/AgendaScreen.kt

```diff
@@ -110,37 +111,7 @@ fun AgendaScreen(
         }
     }
 
-    // Offline? Show delayed warning in case starting up
-    val isOfflineBannerVisible = remember { mutableStateOf(false) }
-    LaunchedEffect(connectivityState) {
-        isOfflineBannerVisible.value = false
-
-        if (connectivityState == IInternetConnectivityObserver.OnlineStatus.OFFLINE
-        ) {
-            delay(1000)
-            isOfflineBannerVisible.value = true
-        }
-    }
-    AnimatedVisibility(isOfflineBannerVisible.value) {
-        Box(
-            modifier = Modifier
-                .fillMaxSize()
-                .background(Color.Transparent)
-        ) {
-            Text(
-                text = stringResource(R.string.no_internet),
-                color = Color.Black,
-                textAlign = TextAlign.Center,
-                fontWeight = FontWeight.Bold,
-                modifier = Modifier
-                    .align(Alignment.TopCenter)
-                    .background(Color.Red.copy(alpha = .5f))
-                    .fillMaxWidth()
-                    .align(Alignment.Center),
-
-                )
-        }
-    }
+    ShowInternetAvailabilityIndicator(connectivityState)
```

Comment:
Is this a normal composable? If yes, shouldn't it rather be called `InternetAvailabilityIndicator` since a composable function is not really an action that's done?

Reply:
Yes this makes sense, I will rename for clarity.